commons/get-help/faq.md

```diff
@@ -121,3 +121,9 @@ can commit them in your application's Clever Cloud repository and then add the
 `clevercloud/ssh.json`.
 
 The ssh.json file is documented in the [common configuration page](/clever-cloud-overview/common-application-configuration/#private-ssh-key)
+
+## I got an `Unsupported major.minor version` error. How can I fix it ?
```

Comment:
I got a `Unsupported major.minor version` error on a java application. How can I fix it?